programs/wrapper/src/market_info.rs

```diff
@@ -24,9 +24,12 @@ pub struct MarketInfo {
     /// Withdrawable quote balance on the market.
     pub quote_balance: QuoteAtoms,
 
+    /// Quote volume traded over lifetime, can overflow.
+    pub quote_volume: QuoteAtoms,
+
     /// Last slot that a sync was called on.
     pub last_updated_slot: u32,
-    pub _padding: [u32; 1],
+    pub _padding: [u32; 3],
```

Comment:
Why this increase? If it is just for uniformity with the core block sizes, make a comment about that.

Reply:
it's the 16 byte alignment cant just add 4 bytes

Reply:
I thought we only need 8 because got rid of u128. Doesn't matter. It is the wrapper. Don't really care about space there

Reply:
well i think you added the 16 byte alignment assert right below the struct definition and that's for a good reason, i think it's fine to keep it to 16b.

Reply:
Don't assume I had a good reason. Haha. Doesn't matter